scripts/pretrain/imagenet-100/ssley.yaml

```diff
@@ -0,0 +1,81 @@
+defaults:
+  - _self_
+  - augmentations: ssley.yaml
```

Comment:
You already have the augmentations below, either remove this or move the augmentations to a separate file (both are fine)

Reply:
Good spot - a heads up I think this is also the case for scripts/pretrain/imagenet-100/vicreg.yaml in that case.

This method should plug in wherever VICReg is used. In many ways the benefit of the loss function is it's got a lot of the nice properties of VICReg but without needing to tune the three loss function parameters.

Reply:
No idea why VICReg is like that haha. I'm fine with having a separate file for the augmentations or having it merged with the main file. Should also prob fix VICReg after that.